pkg/parse/root.go

```diff
@@ -365,14 +365,7 @@ func setRenderingStatus(rendering *v1beta1.RenderingStatus, p Parser, newStatus
 func (p *root) SetSyncStatus(ctx context.Context, errs status.MultiError) error {
 	p.mux.Lock()
 	defer p.mux.Unlock()
-	var allErrs status.MultiError
-	remediatorErrs := p.remediator.ConflictErrors()
-	for _, e := range remediatorErrs {
-		allErrs = status.Append(allErrs, e)
-	}
-	// Add conflicting errors before other apply errors.
-	allErrs = status.Append(allErrs, errs)
```

Comment:
Why do we move this block outside of this function?

Reply:
I've moved the aggregation of sync errors into the updater's new `Errors()` method, wrapped by `Parser.SyncErrors()`. This way it includes all the different sync errors consistency and in once place, instead of having the merges spread across the code.

Reply:
ACK